internal/tracegen/config.go

```diff
@@ -47,6 +48,7 @@ func (c *Config) Flags(fs *flag.FlagSet) {
 	fs.StringVar(&c.Service, "service", "tracegen", "Service name prefix to use")
 	fs.IntVar(&c.Services, "services", 1, "Number of unique suffixes to add to service name when generating traces, e.g. tracegen-01 (but only one service per trace)")
 	fs.StringVar(&c.TraceExporter, "trace-exporter", "otlp-http", "Trace exporter (otlp/otlp-http|otlp-grpc|stdout). Exporters can be additionally configured via environment variables, see https://github.com/jaegertracing/jaeger/blob/main/cmd/tracegen/README.md")
+	fs.StringVar(&c.MarshalToFile, "marshal-to-file", "", "Filename to write generated spans to (in JSON format) instead of sending to a collector")
```

Comment:
the word "marshal" is irrelevant to the purpose of the flag

```suggestion
	fs.StringVar(&c.MarshalToFile, "export-to-file", "", "Filename to write generated spans to (in JSON format) instead of sending to a collector")
```